services/user.py

```diff
@@ -0,0 +1,46 @@
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
```

Comment:
Consider using Django's `create_user` method, which is specifically designed for creating user objects and handles password hashing automatically. This is already being used, so ensure that any additional user creation logic aligns with Django's user model best practices.